### DISCUSSION TITLE
posts/xiaomi14/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#33] [Comments]
# posts/xiaomi14/

俺的华为 Mate 30 Pro，从 2019 年 11 月 11 日购买至今，兢兢业业，于 2023 年 11 月 5 日正式退役。
[&hellip;] 旧手机屏幕之前就被摔过，仍在使用。前两天在我尝试自行车兔跳的时候它从裤兜里飞出，拍在了柏油马路上，这下碎得更加彻底。即便如此，在水凝膜的强力包裹下，仍丝毫不影响使用。
[&hellip;] 对于使用什么样的手机俺并不挑剔，能看到消息有个音儿就 &hellip;

https://shitao5.org/posts/xiaomi14/

**Comment by CyrusYip** [comment]
> 一是发现「小米换新」似乎对从华为搬家到小米使不上劲；

之前试过在荣耀 10 手机用小米换机，相册和通讯录是可以迁移的，但是应用数据不行（同样是 MIUI 系统才行）。微信消息其实可以在旧手机直接传到新手机。

**Reply by Shitao5** [reply]
小红书上看到有人用小米换新把数据都搞丢了，不敢轻易涉险。应用数据好多都和账号绑定，不是什么大问题。微信消息直接传等俺下次换手机再试试哈哈。

**Reply by CyrusYip** [reply]
> 小红书上看到有人用小米换新把数据都搞丢了，不敢轻易涉险。

是旧手机数据丢了，也没传到新手机？那就太惨了。我自己用了两三次小米换机，倒是没出问题。

**Reply by Shitao5** [reply]
是的，那条小红书已经找不到了，那个人是旧手机存不下照片了特地换的手机，结果整没了，想想都很难过。

> 我自己用了两三次小米换机，倒是没出问题。

这是换了三四台手机了嘛，是因为手机卡还是什么呀？我计划用个四五年有戏吗？😂

**Reply by CyrusYip** [reply]
小米换机其实主要是帮家人弄的，家里的智能设备都是我管。我现在用 LineageOS，也用不了小米换机。我最近换机是因为手机又卡电池又不耐用。

小米数字系列性能都是顶尖，用四五年肯定没问题。主要问题是系统更新和电池。系统停更了就没那么安全，到时可以改用开源的第三方安卓系统。电池不耐用就比较烦，现在手机太精细了，自己拆机换电池不容易。

**Reply by Shitao5** [reply]
> 小米换机其实主要是帮家人弄的，家里的智能设备都是我管。

原来如此，家里人很幸福的嘞！

> 电池不耐用就比较烦，现在手机太精细了，自己拆机换电池不容易。

电池我感觉倒还好，可能和个人使用场景有关系。我工作日每天使用时长 3 小时左右，周末就更少了，现在手机充电也越来越快了，对我来说电池容量下降得不要太离谱一般都扛得住。

**Comment by CyrusYip** [comment]
诗涛，我想请你用小米14帮我测试下能不能安装 Google Play，需要在中文语言和英文语言下测试。你可以选择不帮忙，不必有压力。

教程链接：

- [小米手机/红米手机/MIUI Google Play 安装教程 - 叶寻的博客](https://cyrusyip.org/zh-cn/post/2023/05/05/google-play-miui/)
- [How to Install Google Play On Xiaomi Phones / Redmi Phones / MIUI China - Cyrus Yip's Blog](https://cyrusyip.org/en/post/2023/05/05/google-play-miui/)

我想知道操作流程是不是还和我写的一样，不知道设置选项和应用商店APP的名字有没有变化。如果不一样了，麻烦你写一下。还需要你写一下 HyperOS 的版本。先行谢过。

**Reply by Shitao5** [reply]
叶寻你好，以下是我的测试结果：

- 中文版可以安装，第一步有差别：设置 -> 更多设置 -> 帐号与同步 -> 谷歌基础服务。第二步可以直接在应用商店下载安装。
- 英文版也可以安装，同样是第一步的差别：Settings -> Additional settings -> Account & sync -> Basic Google Services。第二步也可以直接在 GetApps 下载安装。

我的 HyperOS 版本：1.0.19.0.UNCCNXM

目前安装 Google Play 完毕打开后提示无法登录，应该是我手机没有梯子的缘故，等我日后整了梯子再来反馈是否可以使用。

**Reply by CyrusYip** [reply]
感谢！我已更新教程，还在文末致谢你了😊。

**Reply by Shitao5** [reply]
叶寻你好，今天我搭了梯子，Google Play 可以登录并使用。
